src/schema/v2/conversation/index.ts

```diff
@@ -323,11 +323,17 @@ export const ConversationType = new GraphQLObjectType<any, ResolverContext>({
             userId: from_id,
           })
 
+          // Assume a conversation only has one item
+          const artwork = items.find((item) => item.item_type === "Artwork")
+          const artworkID = artwork ? artwork.properties.id : null
```

Comment:
We no longer support inquiries on shows, but there is still a handful of conversations without artworks. When those are encountered, I think the `/collector_profile_summary` endpoint will raise an error, which Metaphysics will kick up to the client.

Does that check out with you?

Reply:
From what I got that's the intention, it's not gonna fully break the GraphQL query but will throw an error to this field, right?

Reply:
OK. I've seen GraphQL queries return `null` for fields because the API request behind it has failed, so I guess something like that will happen. (And it will be up to the client to figure out how to handle that `null` field.)

Reply:
Yea - we can handle this a bit better by more 'cleanly' simply returning `null` for show inquiries w/o throwing an error, _or_ throwing a more meaningful error.

However - the idea of throwing an error from a resolver _is_ something we should be doing and we do support it fully. The field returns `null`, _and_ the `errors` are included so a client can do whatever it wants. We _used_ to have an overly-aggressive `reject` on _any_ `errors` setup in clients -> which winds up being too brittle for the client and a possible reason to _not_ throw errors, historically.

But, we've changed that a long time ago, so I think it's actually 'good' to have resolvers throw errors as needed - it's more proper.

That being said - I'll merge this for now but should definitely follow up to handle that case a bit better.